src/components/Tabs/Tabs.spec.tsx

```diff
@@ -0,0 +1,170 @@
+        cy.get("[data-test-id=tab-overflow]").as("OverflowBtn");
+        cy.get("@OverflowBtn").should("be.visible");
+        cy.get("@OverflowBtn").click();
+        cy.get("[role=dialog]").as("Menu");
+        cy.get("@Menu").should("be.visible");
+        cy.get("@Menu").children().last().should("contain.text", data[5].label);
+        cy.viewport(1280, 850);
+        cy.get("@Tabs").children("[data-test-id=tab-overflow]").should("not.exist");
+    });
+
+    it("should not be clickable if disabled", () => {
+        cy.get("[data-test-id=tab-item]").first().click();
+        cy.get("[data-test-id=tab-content]").children().not(".tw-hidden").should("contain.html", data[0].children);
+        cy.get("[data-test-id=tab-item]").eq(1).click();
+        if (data[1].disabled) {
+            cy.get("[data-test-id=tab-item]").eq(1).should("have.class", "tw-text-text-disabled");
+            cy.get("[data-test-id=tab-content]").children().not(".tw-hidden").should("contain.html", data[0].children);
+        } else {
+            cy.get("[data-test-id=tab-item]").eq(1).should("not.have.class", "tw-text-text-disabled");
+        }
+    });
+
+    it("should have Keyboard navigation", () => {
+        cy.get("[data-test-id=tab-item]").first().type("{rightArrow}");
+        cy.get("[data-test-id=tab-item]").eq(2).should("have.class", "tw-font-medium");
+        cy.get("[data-test-id=tab-content]").children().not(".tw-hidden").should("contain.html", data[2].children);
+        cy.get("[data-test-id=tab-item]").eq(2).type("{rightArrow}");
+        cy.get("[data-test-id=tab-item]").eq(3).should("have.class", "tw-font-medium");
+        cy.get("[data-test-id=tab-content]").children().not(".tw-hidden").should("contain.text", "content for tab 4");
+        cy.get("[data-test-id=tab-item]").eq(3).type("{leftArrow}");
+        cy.get("[data-test-id=tab-item]").eq(2).should("have.class", "tw-font-medium");
+        cy.get("[data-test-id=tab-content]").children().not(".tw-hidden").should("contain.text", data[2].children);
+    });
+
+    it("should have Keyboard navigation in overflow menu", () => {
+        // TODO and WIP find a better way to test accessibilty with cypress.
+        cy.viewport(350, 850);
+        cy.get("[data-test-id=tab-overflow]").as("OverflowBtn");
+        cy.get("@OverflowBtn").type("{enter}");
+        cy.get("button#tab-4-btn").click();
+        cy.get("button#tab-4-btn").type("{rightArrow}");
+        cy.get("[data-test-id=tab-content]").children().not(".tw-hidden").should("contain.text", "content for label 6");
+
+        cy.get("@OverflowBtn").type("{enter}");
+        cy.get("button#tab-6-btn").click();
+        cy.get("button#tab-6-btn").type("{leftArrow}");
+        cy.get("[data-test-id=tab-content]").children().not(".tw-hidden").should("contain.text", "content for tab 4");
+    });
+
+    it("should have focus on tab and content", () => {
+        // TODO and WIP find a better way to test accessibilty with cypress.
```

Comment:
We use [cypress-real-events](https://github.com/dmtrKovalenko/cypress-real-events), you can get inspiration from other tests files like [the OrderableList component tests](https://github.com/Frontify/arcade/blob/main/src/components/OrderableList/OrderableList.spec.tsx)